common/CommunityToolkit.Labs.Shared/Renderers/Markdown/MarkdownTextBlock.cs

```diff
@@ -16,11 +16,79 @@
     #endif
 #endif
 
+#if __WASM__
+using Markdig;
+using Uno.Foundation.Interop;
+#endif
+
 namespace CommunityToolkit.Labs.Shared.Renderers;
 
 /// <summary>
 /// Provide an abstraction around the Toolkit MarkdownTextBlock for both UWP and WinUI 3 in the same namespace (until 8.0) as well as a polyfill for WebAssembly/WASM.
 /// </summary>
+#if __WASM__
+public partial class MarkdownTextBlock : Control
+{
+    public string Text
+    {
+        get { return (string)GetValue(TextProperty); }
+        set { SetValue(TextProperty, value); }
+    }
+
+    public static readonly DependencyProperty TextProperty =
+        DependencyProperty.Register(nameof(Text), typeof(string), typeof(MarkdownTextBlock), new PropertyMetadata(null, MarkdownTextChanged));
+
+    private static void MarkdownTextChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+    {
+        if (d is MarkdownTextBlock mtb && mtb.IsLoaded)
+        {
+            mtb.UpdateText(e.NewValue as string ?? string.Empty);
+        }
+    }
+
+    public MarkdownTextBlock()
+    {
+        Loaded += this.MarkdownTextBlock_Loaded;
+    }
+
+    #nullable enable
```

Comment:
@Arlodotexe not sure why I need this here, but VS was giving me a squiggle without it.